doc/testing.md

```diff
@@ -600,14 +600,15 @@ It is highly recommended to use the latest NSS version when running PKCS11
 tests. Improper NSS version may lead to unexpected failures which are hard to
 diagnose. For example, sun/security/pkcs11/Secmod/AddTrustedCert.java may fail
 on Ubuntu 18.04 with the default NSS version in the system. To run these tests
-correctly, the system property `test.nss.lib.paths` is required on Ubuntu 18.04
-to specify the alternative NSS lib directories.
+correctly, the system property `jdk.test.lib.artifacts.<NAME>` is required on Ubuntu 18.04
+to specify the alternative NSS lib directories. The `<NAME>` component should be replaced
+with the name of the appropriate `@Artfact` class. (See `test/jdk/sun/security/pkcs11/PKCS11Test.java`)
```

Comment:
Could you please adjust the line length to match the surrounding text.

When changing any `doc/*.md` file, you will need to regenerate the corresponding html files and include that update in the commit. This can be done with the following make target if you have an appropriate pandoc configured.
```
make update-build-docs
```